Features/Users/users.feature

```diff
@@ -67,3 +67,10 @@ Feature: Use the eZ Users field
         And I fill in "Password" with "12345"
         And I fill in "Confirm password" with "123456"
         Then I should see error messages
+
+    @javascript @edge
+    Scenario: Validate that users have the "Delete" button available
+        Given I am on the Users page
```

Comment:
i tend to agree with @bdunogier. to testing this delete button existence there is no need to check where are you coming front. in fact, you could access this kind of `pages` from other places and button should be there too :)